### ISSUE TITLE
Custom audio menu in the Media Library

### ISSUE THREAD

**Issue by maciekberry** [UniversalMediaServer/UniversalMediaServer#1724]
Hello UMS-masters,

If you try to manage a fairly large music library with UMS (let's say hundreds of albums), you quickly realize that most of the options available from the "Media Library" folder are totally useless. I see no real-world example when one would like to display all the songs (thousands) at once. Or all Albums, without having them sorted by Artist. 

To make long story short, please check the following commit to my fork of UMS:
https://github.com/maciekberry/UniversalMediaServer/commit/5ab5687a1039dd9b859935b52fc19de583289d46

It's a quick'n'dirty shoot at the problem. It adds a new variable to UMS.conf (audio_menu_tree).
It contains a ordered list of items the user wants to make appear under his/her media library. It is read and used while creating the tree.

I am currently testing this at home and well, it works :) 

Of course it is totally not-ready for prime time: it's not really that robust, there no default value provided etc; and of course it should be extended to Video and Picture as well. Perhaps it could also include more options (like disabling Audio or Video or Picture alltogether).

If you are interested in this feature in mainstream, let me know, I could try to finish it and make it production-ready after we discuss how it should ideally work.

Cheers,
Maciek Berry

**Comment by SubJunk** [comment]
@maciekberry yes this sounds great. I would be happy for there to be more useful defaults in there, and the ability for users to customize it. In the future we can even add a GUI for that.
